tests/system/providers/google/cloud/transfers/example_gcs_to_sftp.py

```diff
@@ -46,12 +53,35 @@
     catchup=False,
     tags=['example', 'gcs'],
 ) as dag:
+    create_bucket = GCSCreateBucketOperator(
+        task_id="create_bucket", bucket_name=BUCKET_NAME, project_id=PROJECT_ID
+    )
+
+    upload_file_1 = LocalFilesystemToGCSOperator(
```

Comment:
wondering why you are uploading multiple times

Reply:
To ensure I have file available on different paths - directly in bucket root, under directory etc. It's łatwe used by transfer operator and we are testing if its possible to copy file, copy file from dir (and recreate dir structures), copy with wildcard. But thanks to your question I have noticed we dont need third upload - since 2nd upload already puts file under dir and we came reuse it.

Reply:
I realized one of the operators has a "move_object" parameter which will delete files from the source directory. That's why we need all three separate uploads to cover all test scenarios:

Copy single file - 1st upload
Move file from the directory (removing source) - 2nd upload
Copy file from the directory when preserving dir structure - 3rd upload
Copy file from the directory without preserving dir structure - 3rd upload